Gems/WarehouseAutomation/gem.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Please take a look at Gem versioning and version dependencies (a recent feature), for example in other Gems in o3de-extras.